examples/plugins/calculator/main.go

```diff
@@ -4,22 +4,34 @@ import (
 	"strconv"
 
 	"github.com/aerogear/charmil/pkg/factory"
+	"github.com/aerogear/charmil/pkg/localize"
 	"github.com/aerogear/charmil/pkg/logging"
 	"github.com/spf13/cobra"
+	"golang.org/x/text/language"
 )
 
 type Options struct {
-	Logger func() (logging.Logger, error)
+	Logger   func() (logging.Logger, error)
+	Localize localize.Localizer
 }
 
-func RootCommand(f *factory.Factory) *cobra.Command {
+func RootCommand() *cobra.Command {
+
+	loc, err := localize.InitLocalizer(localize.Config{Language: language.English, Path: "examples/plugins/calculator/locals/en/en.yaml", Format: "yaml"})
+
+	if err != nil {
+		panic(err)

```

Comment:
Panic is not going to provide good output. Never use panic.
Especially that logger is there ( However undocumented )

Reply:
Okay 👍
Have to add some more methods in logger. Will document it.

Reply:
Example should be smaller. 

If we basically copy paste other library example and implement it what is the value we provide?

Reply:
will not say it exact copy paste, but yeah it is. When you see a good approach in the front of you, it blocks the way for other things. But I will try to minimize these things in further contributions!